tests/testdata/sklearn_transform_parameters.json

```diff
@@ -2,5 +2,5 @@
   "numeric_imputer_strategy": "constant",
   "numeric_imputer_constant_fill": -50.0,
   "numeric_standardize_with_mean": 0,
-  "categorical_fill": "unicode compatibility: 🚀"
+  "categorical_fill": "unicode compatibility: \uD83D\uDE80🚀"
```

Comment:
what's this change for?

Reply:
weird, the unicode didn't render for me locally. I'll revert this file